program_v2/programs/sol-did/src/security_txt.rs

```diff
@@ -0,0 +1,14 @@
+pub mod security {
+    use solana_security_txt::security_txt;
+
+    security_txt! {
+        // Required fields
+        name: "Sol-DID",
+        project_url: "https://github.com/identity-com/sol-did",
+        contacts: "email: contact@identity.org",
+        policy: "https://docs.identity.com/docs/sol-did/#policies",
+        encryption: "https://docs.identity.com/docs/sol-did/#encriptions",
```

Comment:
spelling

Reply:
Plus, I think this should point to the PGP key directly and not an anchor within a document